Library/ViewModels/RewardAddOnSelectionViewModel.swift

```diff
@@ -256,18 +256,79 @@ public final class RewardAddOnSelectionViewModel: RewardAddOnSelectionViewModelT
     .map(PledgeViewData.init)
     .takeWhen(self.continueButtonTappedProperty.signal)
 
+    let calculatedShippingTotal = Signal.combineLatest(
+      shippingRule.skipNil(),
+      allRewards,
+      selectedQuantities
+    )
+    .map(calculateShippingTotal)
+
+    let shippingTotal = Signal.zip(project, baseReward)
```

Comment:
This var should probably be called `baseRewardShippingTotal` based on the name of the method being used.